website/app/styles/doc-components/icons-list/index.scss

```diff
@@ -103,10 +109,36 @@
 }
 
 .doc-icons-list-grid-item__meta {
+  position: relative;
   width: 100%;
-  padding: 16px;
+  min-width: 0;
+  padding: 8px 16px;
 }
 
+.doc-icons-list-grid-item__permalink {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
The `align-items` & `justify-content` styles aren't having an effect. Add `display: flex;` to make them work.